subsys/bluetooth/audio/tbs.c

```diff
@@ -111,10 +112,10 @@ struct tbs_inst {
 static struct tbs_inst svc_insts[CONFIG_BT_TBS_BEARER_COUNT];
 static struct tbs_inst gtbs_inst;
 
-#define READ_BUF_SIZE                                                                              \
-	MAX(BT_ATT_MAX_ATTRIBUTE_LEN,                                                              \
-	    (CONFIG_BT_TBS_MAX_CALLS * sizeof(struct bt_tbs_current_call_item) *                   \
-	     (1U + ARRAY_SIZE(svc_insts))))
+#define READ_BUF_SIZE                                                                             \
+	MAX(BT_ATT_MAX_ATTRIBUTE_LEN,                                                             \
+	    (CONFIG_BT_TBS_BEARER_COUNT * CONFIG_BT_TBS_MAX_SCHEME_LIST_LENGTH +                  \
+	     CONFIG_BT_TBS_MAX_SCHEME_LIST_LENGTH))
```

Comment:
Hmm, I think this calculation need to take both the previous definition and the new `CONFIG_BT_TBS_MAX_SCHEME_LIST_LENGTH` handling into account. 

So it probably should be something like

```suggestion
	    MAX((CONFIG_BT_TBS_MAX_CALLS * sizeof(struct bt_tbs_current_call_item) * \
	    (1U + ARRAY_SIZE(svc_insts))), \ 
	    (CONFIG_BT_TBS_BEARER_COUNT * CONFIG_BT_TBS_MAX_SCHEME_LIST_LENGTH +                  \
	     CONFIG_BT_TBS_MAX_SCHEME_LIST_LENGTH))
```

right?

Reply:
agree, I will update the code as suggested. thank you :)